applications/StructuralMechanicsApplication/tests/test_rve_analytic.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import print_function, absolute_import, division
+
+import KratosMultiphysics
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+import KratosMultiphysics.kratos_utilities as kratos_utilities
+
+import RVEAnalysis
+
+import os
+
+class controlledExecutionScope:
```

Comment:
please use the new class in `kratos_utilities` for this

Reply:
@philbucher i am not clear what you are asking for. could u point me to an example?

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/master/applications/StructuralMechanicsApplication/tests/structural_mechanics_test_factory.py#L13

Reply:
thx, i think i corrected this